### PR TITLE
#58 setup google slides addon project

### PR DESCRIPTION
**Pull request by Max44444**

### CODE REVIEW

File: slidez-fe-addon/README.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
+<p align="center">
```
Comment (Anatolii-Ivanov) [inline slidez-fe-addon/README.md:1]: Are you sure this file is needed?

File: slidez-fe-addon/src/client/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Client (our React code)
```
Comment (Anatolii-Ivanov) [inline slidez-fe-addon/src/client/README.md:1]: Check if all Readme is needed

File: slidez-fe-addon/src/client/sidebar-demo/index.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (Anatolii-Ivanov) [inline slidez-fe-addon/src/client/sidebar-demo/index.html:1]: It's better to give correct names for files, as it seems to be  just a sidebar later, but it FYI

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]
Add package-lock.json to gitignore

**Review by klesogor — APPROVED** [review]

**Review by NDMyroshnychenko — APPROVED** [review]

**Review by Max44444 — COMMENTED** [review]
.

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**